### ISSUE TITLE
Candybar Thread

### ISSUE THREAD

**Issue by Aman9das** [Donnnno/Arcticons#314]
Discussion about what is to be done now that code-base will move to candybar..

**Comment by Aman9das** [comment]
I think we should wait a bit before releasing on candybar.. 

- polish the theme to suit arcticons,
- use the font that is used in the icons in the app,
- figure out how icon requests will be handled, i.e. how the community will see the requested icons..
- a general roadmap maybe?

**Comment by Donnnno** [comment]
> polish the theme to suit arcticons,

Yea, that and write additional texts where needed.

> use the font that is used in the icons in the app,

I probably should tweak that font a bit, there are some kerining issues, so in the meantime I used Inter as the main font 

> figure out how icon requests will be handled, i.e. how the community will see the requested icons..

I've set up an email for that: arcticons@onnno.nl and I want to upload those requests to Github.

>a general roadmap maybe?

Created a milestone for this (never used that before, but should be fun).

**Comment by Donnnno** [comment]
You can view all the issues in the milestone now.

**Comment by Donnnno** [comment]
Nope, my real name is Onno. So, I bought onnno.nl a while ago because onno.nl wasn't available :^)

**Comment by Donnnno** [comment]
@Aman9das a small thing when working on candybar: for now, keep new icons separated from everything else candybar related. Then it will be easier to keep track of all the todo's, with the new icons as the finishing touch when everything's ready!

**Comment by Aman9das** [comment]
Yes you're right.. go ahead and clear the icon list except a few -firefox, bromite, fennec and chrome.. so we still have some form of reference for the icons and we can focus more on the important stuff..

**Comment by Donnnno** [comment]
Nah, it's fine as it is now. But if we change it constant it can get confusing haha

**Comment by Aman9das** [comment]
https://youtu.be/hD7xWRfpHgA
https://fire.fundersclub.com/

We can set this bot up to relay emails to github issues!

**Comment by Donnnno** [comment]
Fuck it, clock widget:

![image](https://user-images.githubusercontent.com/31142286/132949714-071ec722-453f-49df-baed-15c12dd30eeb.png)

**Comment by Aman9das** [comment]
Does it not require kgww or something to work !?

It no then this is amazing, another bird down..

**Comment by joelchrono12** [comment]
> Fuck it, clock widget:
> 
> ![image](https://user-images.githubusercontent.com/31142286/132949714-071ec722-453f-49df-baed-15c12dd30eeb.png)

sacrilegious xD.
Looks kinda good, can you actually customize that when building the app

**Comment by Donnnno** [comment]
> kgww

? What's that, I didn't activate anything. Just put 3 png's in a drawable folder.

**Comment by Donnnno** [comment]
> can you actually customize that when building the app

Yup.

**Comment by joelchrono12** [comment]
> kgww

This is KWGT, its an app that allows you to create custom widgets. Its one of the most powerful customization apps on Android.
It is paid tho. But its worth it.

**Comment by Aman9das** [comment]
@Donnnno could you add Discussions to the repo... its at Settings > Options > Features > Discussions
https://docs.github.com/en/discussions/quickstart

So that its easier to ask questions..

**Comment by Donnnno** [comment]
Done :)

**Comment by Donnnno** [comment]
I Think we're pretty close to finishing the candybar migration

**Comment by joelchrono12** [comment]
> I Think we're pretty close to finishing the candybar migration

Release today? :^)

**Comment by Donnnno** [comment]
Nah, but this week, maybe. Looking at the Arcticons release schedule, it's kinda crazy how fast everything goes. Like almost every week ~50 icons :)

**Comment by joelchrono12** [comment]
The power of free software

**Comment by Donnnno** [comment]
I think I may be able to finish everything tomorrow, and then I have to figure out the best way to merge everything to main without fucking everything up.

**Comment by Aman9das** [comment]
Hey what's the name of the font we use on the icon-pack, if its your own custom font, then we can help you make a ttf file using forgefont.

Also would it ok to have a Digital clock that isn't resizable? I cant figure out how to make the font size resizable.

As for merging I think it'll be alright, pretty excited!

**Comment by Donnnno** [comment]
> if its your own custom font, then we can help you make a ttf file using forgefont.

I've actually done that already in the past, it's not perfect tho because kerning takes ages to do right, you can try it [here](https://www.onnno.nl/lab.html ), I'll export a TTF file later but beware, it's not perfect and has only one weight. 

Quicksand is a nice alternative. 

Is it possible to have two widgets in there? I like the analogue clock version too!

**Comment by Donnnno** [comment]
Here's the ttf file 

[Arcticons-Regular.zip](https://github.com/Donnnno/Arcticons/files/7152626/Arcticons-Regular.zip)

**Comment by Aman9das** [comment]
![Screenshot_20210913_135411.jpg](https://user-images.githubusercontent.com/39594914/133049630-3f6aef83-4726-4d90-aa01-1427eeac7f58.jpg)

Both the widgets co-existing :)

**Comment by Donnnno** [comment]
Great! Let's do that then :)

**Comment by Donnnno** [comment]
All icons, including the new ones, are rasterized now for both dark & light modes!

**Comment by joelchrono12** [comment]
> All icons, including the new ones, are rasterized now for both dark & light modes!

Does that mean that there wont be PNG images used now?

**Comment by Donnnno** [comment]
Rasterizing means making raster images (PNG/JPEG) from vectors (SVG/XML), so we still use PNG's but in a higher resolution than before.

**Comment by Aman9das** [comment]
Hey i forgot.. next time we gotta use higher definition icons (400px) because i saw icon packs based on candybar use it

**Comment by Donnnno** [comment]
Ehh, realistically speaking that's overkill for almost every device out there, and for the ones that do have a high enough resolution display, you don't really see it.

**Comment by Donnnno** [comment]
The importance of making sure everything is white/black:

![Screenshot_20210913-152253_Arcticons_Light.png](https://user-images.githubusercontent.com/31142286/133091365-eca4a6fc-1cfe-4e5d-b0d4-a51bc7e4bf4f.png)

When an icon isn't #ffffff it can get converted the wrong way. 🙃

**Comment by Aman9das** [comment]
Can you tell where the problem occured, at the contributor's svg or in the middle of the script..?

**Comment by Donnnno** [comment]
At the contributor's SVG. And then I probably didn't see it while checking it, stuff like that sometimes happens, will fix that later today.

**Comment by joelchrono12** [comment]
> At the contributor's SVG. And then I probably didn't see it while checking it, stuff like that sometimes happens, will fix that later today.

I have a question regarding this. I always have a feeling that some icons have lines that are not 1px (or 1pt) of weight. Is there a way to check that in code? Maybe its just me, but man it bothers me when an icon is slightly more bold than others

**Comment by Donnnno** [comment]
Yea, I'm thinking that too sometimes but when I check the SVG's they say it's 1px :s

**Comment by Donnnno** [comment]
**WHITE:**
Tunerly
Usagedirect
Unchained

**BLACK:**
Alitelecom
Androbd
Anutotd
Appbrainaddetector
Athmvil
Battlelands
Bricksbreakerballcrush
Catima
Colornote
Concentricclock
Discord
Frozenbubble
Jiocinema
Maingau
Nanji
Nonocross
Note
Notebloc
Obscuracam
Plugsurfing
Projectm
Proshot
Rpur
Runnerup
Secretcodes
Seedship
Tinfoilforfacebook
Trussme
Tunerly
Unchained
Usagedirect
Whereby
Windy
Xscreensaver

**Comment by joelchrono12** [comment]
What?

**Comment by Donnnno** [comment]
This: 
https://github.com/Donnnno/Arcticons/commit/58666e6035f938353a35343aa32cada65e8084f9

**Comment by Aman9das** [comment]
Created the pr with the desired changes, i think it looks pretty good..

**Comment by Donnnno** [comment]
Nice! 

With that, I think we're almost ready to push this update!

What do you think @Aman9das ?

**Comment by Aman9das** [comment]
Yes of course.. otherwise it'll stay in development forever xp

**Comment by Donnnno** [comment]
And now we play the F-Droid waiting game, where we'll have to see if everything builds correctly with this new dashboard.

**Comment by Aman9das** [comment]
Close this issue, now that candybar is functioning?
